discord/time.go

```diff
@@ -138,3 +147,34 @@ func (ms Milliseconds) Duration() time.Duration {
 	const f64ms = Milliseconds(time.Millisecond)
 	return time.Duration(ms * f64ms)
 }
+
+//
+
+// ArchiveDuration is the duration after which a thread without activity will
+// be archived.
+//
+// The duration is counted in minutes.
+type ArchiveDuration int
+
+const (
+	OneHourArchive ArchiveDuration = 60
+	OneDayArchive  ArchiveDuration = 1440
+	// ThreeDayArchive archives a thread after three days.
+	//
+	// This duration is only available to nitro boosted guilds. The Features
+	// field of a Guild will indicate whether this is the case.
+	ThreeDayArchive ArchiveDuration = 4320
+	// SevenDayArchive archives a thread after seven days.
+	//
+	// This duration is only available to nitro boosted guilds. The Features
+	// field of a Guild will indicate whether this is the case.
+	SevenDayArchive ArchiveDuration = 10080
```

Comment:
`SevenDayArchive` and `ThreeDayArchive` should be in plural form: `SevenDaysArchive` and `ThreeDaysArchive`.